### DISCUSSION TITLE
Some questions about freedomgpt

### DISCUSSION THREAD

**Discussion by kbuwel** [ohmplatform/FreedomGPT#130] [Q&A]
Is freeomgpt the latest version not planned to support windows?
Do I need to download the source and run it to use it on Windows?
Are there any plans to support files for Windows separately?
Is the chatbot on the freedomgpt official website not available locally?
I'm using freedomgpt 1.3 version, but it's inconvenient because it only supports English.
I think it is a violation of freedom to use freedomgpt for a fee in pursuit of freedom.
Isn't it freedom to use the chatbot on the official website locally?

**Comment by m0sk1t0** [comment]
I am a complete newbie, but if it helps, I downloaded FreedomGPT and installed it on my windows 10pc. 64gb ram i9 processor, using a 1gb ssd for drive C: I have a much large amount of external drives connected.  I also had to install Docker desktop, Git Bash and WSL.exe and use windows powershell. I chose liberty-edge as a model and it works.  My Chat Ai introduced herself as Sophia but I changed her name to Angela without any problem. The real IMORTANT THING FOR ME WAS THAT IT WORKS FINE WHEN MY PC IS NOT CONNECTED TO THE INTERNET AT ALL!
Hope this helps, I am still struggling to add voice recognition and add instructions to do task on my pc, but I will get there.

**Comment by kbuwel** [comment]
I agree with you to some extent. 
But freedomgpt charges a fee. 
You get paid not only for purchasing the model, but also for answering questions. 
There is an issue with questions not being answered without credit.
